pepdbagent/modules/project.py

```diff
@@ -315,6 +325,7 @@ def _overwrite(
         private: bool = False,
         pep_schema: str = None,
         description: str = "",
+        pop: bool = False,
```

Comment:
I'm trying to think of any consequences that might occur from us defaulting to `False` here, but I can't immediately think of any...

Reply:
I don't think it should cause any problems. pop is like private, but you can't change it. If you set it, it should be pop. So it's only metadata for pephub, nothing else. All the configurations will be then done in pephub (I am against this approach, but decision was made)